06_gpu_and_ml/llm-serving/vllm_inference.py

```diff
@@ -33,12 +33,15 @@
 import modal
 
 vllm_image = (
-    modal.Image.debian_slim(python_version="3.12")
-    .pip_install(
-        "vllm==0.9.1",
-        "huggingface_hub[hf_transfer]==0.32.0",
-        "flashinfer-python==0.2.6.post1",
-        extra_index_url="https://download.pytorch.org/whl/cu128",
+    modal.Image.from_registry("nvidia/cuda:12.8.0-devel-ubuntu22.04", add_python="3.12")
+    .uv_pip_install(
```

Comment:
Hmm, doing this in two steps means `pip` doesn't keep all dependencies aligned -- e.g. `vllm` is free to install whatever `torch` version it wants in the next step